From bfe0ffae835dc5769198ae4aa5625ce44076eeb7 Mon Sep 17 00:00:00 2001
From: JP-Ellis <josh@jpellis.me>
Date: Mon, 2 Dec 2019 21:19:46 +1100
Subject: [PATCH] Update supported Python version to 3.5, 3.6, 3.7 & 3.8

It's probably safe to drop support for Python <= 3.4 as they are neither
maintained nor receiving security fixes.

Signed-off-by: JP-Ellis <josh@jpellis.me>
---
 .travis.yml  |  3 +--
 appveyor.yml | 10 ++++------
 setup.py     |  4 ++--
 3 files changed, 7 insertions(+), 10 deletions(-)

diff --git a/.travis.yml b/.travis.yml
index a94caae6..6f73695a 100644
--- setup.py
+++ setup.py
@@ -86,10 +86,10 @@
         'Operating System :: MacOS',
         'Operating System :: POSIX',
         'Operating System :: Unix',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
         'Topic :: Utilities',
     ],
     extras_require=dict(
